### DISCUSSION TITLE
Unit Testing & Code Quality

### DISCUSSION THREAD

**Discussion by rikazrizlan** [EddieHubCommunity/LinkFree#1855] [Ideas]
I went through the repo and didn't find any files related to unit testing, is there a specific reason why?  
I think writing unit tests would be good as it's an open source project and anyone can contribute and also consider configuring the project with SonarQube to have an eye on bugs, code smells, security hotspots and unit test coverage.

**Comment by eddiejaoude** [comment]
We are doing end to end testing using Playwright, we find them the most valuable. The files are in the `tests` folder
